packages/design-system-react/src/components/icon/icons/index.ts

```diff
@@ -0,0 +1,14 @@
+// This file is auto-generated. Do not edit manually
```

Comment:
The `index` file generated by the `generate-icons-index` script was originally ignored in Git, but we need it to be included for tests.